nldi-liquibase/src/main/resources/liquibase/characteristic_data/tables.sql

```diff
@@ -0,0 +1,64 @@
+--liquibase formatted sql
+
+--changeset ayan:create.characteristic_data.characteristic_metadata
+--preconditions onFail:MARK_RAN onError:HALT
+--precondition-sql-check expectedResult:0 select count(*) from information_schema.tables where table_schema = 'characteristic_data' and table_name = 'characteristic_metadata'
+create table characteristic_data.characteristic_metadata
+(characteristic_id              character varying(500) not null
+,characteristic_description     character varying(4000)
+,units                          character varying(4000)
+,dataset_label                  character varying(4000)
+,dataset_url                    character varying(4000)
+,theme_label                    character varying(4000)
+,theme_url                      character varying(4000)
+,characteristic_type            character varying(4000)
+,constraint characteristic_metadata_pk
+    primary key (characteristic_id)
+)
+WITH (OIDS=TRUE);
```

Comment:
My understanding of it is that the syntax has changed, but OIDS are still something that is sanctioned for use. That said, I'm skeptical of there utility in this schema... the values are only four bytes in size and can't guarantee uniqueness once the database reaches a certain size and the values turnover.

I'm going to take it out.